_posts/2015-09-15-meetup.md

```diff
@@ -0,0 +1,57 @@
+---
+title: All-React Extravaganza
+author: aaron
+layout: post
+categories:
+- posts
+- meetups
+when: 2015-09-15T19:30:00-05:00
+---
+
+Webster's Dictionary [defines "extravaganza" as][extravaganza]:
+
+> a very large and exciting show or event
+
+And nothing says exciting like JavaScript frameworksssssss!
+
+<div class="sponsor-logo">
+  <img src="https://cldup.com/D3VCZfiOrY.gif" style="width: 300px; max-width: 100%;" alt="WHOA REACT JS" />
```

Comment:
the animated miami night club seems ok to me, but i'm not an expert.

we could do an acrostic in comic sans instead:
Really
Excellent
Awesome
Cool
Tubular
Jammin
Spectacular